packages/swapper/src/swappers/zrx/ZrxBuildQuoteTx/ZrxBuildQuoteTx.ts

```diff
@@ -145,20 +143,17 @@ export async function ZrxBuildQuoteTx(
       txData: data.data,
       sellAmount: data.sellAmount,
       buyAmount: data.buyAmount,
-      guaranteedPrice: data.guaranteedPrice,
       allowanceContract: data.allowanceTarget,
-      sources: data.sources?.filter((s) => parseFloat(s.proportion) > 0) || DEFAULT_SOURCE,
-      priceImpact
+      sources: data.sources?.filter((s) => parseFloat(s.proportion) > 0) || DEFAULT_SOURCE
```

Comment:
unrelated to this PR - @0xApotheosis we should add the trailing commas lint rule to this repo too to avoid these multi-line diffs